### PR TITLE
Fixing the build break.

### PR DESCRIPTION
**Pull request by sornaks**
Reverting the timing based functional test.

### CODE REVIEW

File: test/WebSites/MvcTagHelpersWebSite/Controllers/Catalog_CacheTagHelperController.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ public class Catalog_CacheTagHelperController : Controller
         public ProductsService ProductsService { get; set; }
 
         [HttpGet("/catalog")]
-        public ViewResult Splash(int categoryId, int correlationId, [FromHeader] string locale)
+        public ViewResult Splash(int categoryId, int correlationId, [FromHeader]string locale)
```
Comment (dougbu) [inline test/WebSites/MvcTagHelpersWebSite/Controllers/Catalog_CacheTagHelperController.cs]: nit: that was a very nice space :smiling_imp:

**Comment by dougbu** [comment]
:shipit:

**Comment by dougbu** [comment]
@sornaks haven't you checked this in?  suggest closing the PR

**Comment by sornaks** [comment]
Yes checked in.

**Pull request closed without merging**